devbin/mujin_webstackclientpy_generategraphclient.py

```diff
@@ -87,6 +87,8 @@ def _DiscoverMethods(queryOrMutationType):
     return methods    
 
 def _PrintMethod(queryOrMutation, operationName, parameters, description, returnType):
+    if queryOrMutation == 'query' and operationName.startswith("List"):
+        print('    @BreakLargeGraphQuery')
```

Comment:
@LLipter the decorator hides the docstrings, can you check if there is a workaround? 

```python
In [1]: self.graphApi.ListEnvironments?
Signature: self.graphApi.ListEnvironments(*args, **kwargs)
Docstring: <no docstring>
File:      /checkoutroot/mujinwebstackclientpy/python/mujinwebstackclient/webstackgraphclientutils.py
Type:      instancemethod
```

Reply:
I found this method

```python
    # relay the docstring
    inner.__doc__ = queryFunction.__doc__
```

Reply:
also https://docs.python.org/2.7/library/functools.html#functools.wraps